interface/src/app/analyze/analysis/analysisDetail.tpl.html

```diff
@@ -1,11 +1,20 @@
 <!--
   TODO each sample should be built using a reusable Angular directive
 -->
-<h1>Samples
-  <small>(drag <i class="fa fa-bars" aria-hidden="true"></i> to reorder,
-    click <i class="fa fa-times" aria-hidden="true"></i> to delete,
-    choose <input type="radio" disabled> to set group)</small>
-</h1>
+
+<div class="page-header">
+  <h1>Samples
+    <small>(drag <i class="fa fa-bars" aria-hidden="true"></i> to reorder,
+      click <i class="fa fa-times" aria-hidden="true"></i> to delete,
+      choose <input type="radio" disabled> to set group)</small>
+  </h1>
+  <ml-model-view></ml-model-view>
+  <ml-model-validator model-id="modelInUrl" is-valid-model="isValidModel">
+  </ml-model-validator>
+</div>
+
+<div ng-if="isValidModel">
```

Comment:
@mhuyck : gotcha - yea - a "sample" should probably tie back to an organism. @kurtwheeler's data refinery should have the organism information stored, so this will be something that i'm pretty sure we can get input